### ISSUE TITLE
Numbers change in Tree Mode

### ISSUE THREAD

**Issue by mahyari** [joeferner/redis-commander#400]
I'm using latest docker image of Redis Commander. When i try to switch to Tree Mode, some long number values changed!
for example below object:
```{ "pi":66110734225681139 }```
change to:
```
{
   "pi": 66110734225681140
}
```
in Tree mode!

**Comment by sseide** [comment]
Thanks for reporting - its a problem with Javascipt and it internal rounding of numbers. 

Short test on command line with node interpreter:
```
$ node
> var x="66110734225681139";
undefined
> console.log(x);
66110734225681139
undefined
> console.log(Number(x));
66110734225681140
```
Not sure what we can do about it or if its a problem with the json tree view library used. 
Will check it after easter hoiday.

**Comment by mahyari** [comment]
Hi,
You are right, but I checked some online JSON parser like [this](http://json.parser.online.fr/) that works right in this situation. I think you it's better to change the viewer because of wrong presented data that may harm someone who don't know about this bug.

**Comment by sseide** [comment]
The one you referenced is a php page and serverside libraries do not help. We need a client side javascript library witch is fast, has no security problems and a fitting license for free use.

I'll add a workaround to display these bigger numbers as strings - is not the very exact same json than but no false values anymore. And i added a PR to the json-view library we use to support handling of big numbers (https://github.com/abodelot/jquery.json-viewer/pull/21). If this is accepted we can get a full working version with all numbers displayed as such.

**Comment by sseide** [comment]
pushed new version that displays big numbers as string, should be available from dockerhub within next time.
No npm relase available.

**Comment by sseide** [comment]
closed with release of version 0.8.0 - now big integer displayed correctly as number.

But there is another related bug #479 as this is not working with big floats
